proc_scripts/plotting.py

```diff
@@ -1,5 +1,8 @@
 from pyspecdata import *
 from sympy import symbols
+import matplotlib.pyplot as plt
+import numpy as np
+logger=init_logging('info')
```

Comment:
again, init logging in the main script.  here, use "logging.info" or "logging.debug"

Reply:
import logging if needed